tests/integration/modifiers/animate-loading-test.js

```diff
@@ -8,36 +8,38 @@ module('Integration | Modifier | animate-loading', function (hooks) {
 
   test('it renders defaults', async function (assert) {
     await render(hbs`<div {{animate-loading}}></div>`);
-    assert.dom('div').hasStyle({
-      opacity: '0.1',
-    });
+    assert.ok(
+      Math.abs(getComputedStyle(this.element.querySelector('div'), null).opacity - 0.1) < 0.001
```

Comment:
:exploding_head:

Reply:
Yup. I'm not super happy about this, but it's in a test so 🤷